<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="2" ignored="0" total="3" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-24T16:14:42 SAST" name="Suite" finished-at="2025-07-24T16:17:49 SAST" duration-ms="187194">
    <groups>
      <group name="Sanity">
        <method signature="RegisterAccountLandingPageTest.registrationAccountTest()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" name="registrationAccountTest" class="com.mystore.testcases.RegisterAccountLandingPageTest"/>
      </group> <!-- Sanity -->
      <group name="Regression">
        <method signature="RegisterAccountLandingPageTest.registrationAccountTest()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" name="registrationAccountTest" class="com.mystore.testcases.RegisterAccountLandingPageTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2025-07-24T16:14:42 SAST" name="Test_All" finished-at="2025-07-24T16:17:49 SAST" duration-ms="187194">
      <class name="com.mystore.testcases.RegisterAccountLandingPageTest">
        <test-method is-config="true" signature="loadConfig()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:14:41 SAST" name="loadConfig" finished-at="2025-07-24T16:14:42 SAST" duration-ms="1006" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadConfig -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:14:42 SAST" name="setup" finished-at="2025-07-24T16:14:56 SAST" duration-ms="14139" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method retried="true" signature="registrationAccountTest()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:14:56 SAST" name="registrationAccountTest" finished-at="2025-07-24T16:15:02 SAST" duration-ms="5518" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Register Account1] but found [Register Account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Register Account1] but found [Register Account]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.mystore.testcases.RegisterAccountLandingPageTest.registrationAccountTest(RegisterAccountLandingPageTest.java:50)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationAccountTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:15:02 SAST" name="tearDown" finished-at="2025-07-24T16:15:15 SAST" duration-ms="12941" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:15:15 SAST" name="setup" finished-at="2025-07-24T16:15:33 SAST" duration-ms="18274" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method retried="true" signature="registrationAccountTest()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:15:33 SAST" name="registrationAccountTest" finished-at="2025-07-24T16:15:44 SAST" duration-ms="11224" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Register Account1] but found [Register Account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Register Account1] but found [Register Account]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.mystore.testcases.RegisterAccountLandingPageTest.registrationAccountTest(RegisterAccountLandingPageTest.java:50)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationAccountTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:15:44 SAST" name="tearDown" finished-at="2025-07-24T16:16:09 SAST" duration-ms="24986" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:16:09 SAST" name="setup" finished-at="2025-07-24T16:17:14 SAST" duration-ms="64979" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="registrationAccountTest()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:17:14 SAST" name="registrationAccountTest" finished-at="2025-07-24T16:17:32 SAST" duration-ms="17850" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Register Account1] but found [Register Account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Register Account1] but found [Register Account]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.mystore.testcases.RegisterAccountLandingPageTest.registrationAccountTest(RegisterAccountLandingPageTest.java:50)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationAccountTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:17:33 SAST" name="tearDown" finished-at="2025-07-24T16:17:49 SAST" duration-ms="16000" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="afterSuit()[pri:0, instance:com.mystore.testcases.RegisterAccountLandingPageTest@3d51f06e]" started-at="2025-07-24T16:17:50 SAST" name="afterSuit" finished-at="2025-07-24T16:18:00 SAST" duration-ms="10112" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuit -->
      </class> <!-- com.mystore.testcases.RegisterAccountLandingPageTest -->
    </test> <!-- Test_All -->
  </suite> <!-- Suite -->
</testng-results>
